xml/depl_maintenance.xml

```diff
@@ -508,6 +508,258 @@
+Running chef-client on d52-54-77-77-01-01.vo6.cloud.suse.de...
+Log: /var/log/crowbar/db-prepare.chef-client.log on
+d52-54-77-77-01-01.vo6.cloud.suse.de Run time: 444.725193199s
+Removing recipe[database::pg2mariadb_preparation] from run_list
+Prepare completed for d52-54-77-77-01-01.vo6.cloud.suse.de
+Summary of used databases: /etc/pg2mysql/databases.yaml on
+d52-54-77-77-01-01.vo6.cloud.suse.de</screen>
+      <para>The <literal>Summary of used databases:</literal> line shows the
+      exact location of the <filename>/etc/pg2mysql/databases.yaml</filename> file.
+      </para>
+      <para>
+       The <filename>/etc/pg2mysql/databases.yaml</filename> file contains a
+       list of databases along with their source and target connection strings:
+      </para>
+<screen>keystone:
+  source: postgresql://keystone:vZn3nfxXzv97@192.168.243.87/keystone
+  target: mysql+pymysql://keystone:vZn3nfxXzv97@192.168.243.88/keystone?charset=utf8
+glance:
+  source: postgresql://glance:cOau7NhaA54N@192.168.243.87/glance
+  target: mysql+pymysql://glance:cOau7NhaA54N@192.168.243.88/glance?charset=utf8
+cinder:
+  source: postgresql://cinder:idRll2gJPodv@192.168.243.87/cinder
+  target: mysql+pymysql://cinder:idRll2gJPodv@192.168.243.88/cinder?charset=utf8</screen>
+     </step>
+     <step>
+      <para>
+       Install the <package>python-psql2mysql</package> package on the
+       &admnode; or directly on the &dbnode;.
+      </para>
+      <para>
+       Running the tool directly on &dbnode; speeds up the migration operation,
+       as this does not require moving database data over the network. You can
+       also use <filename>/etc/pg2mysql/databases.yaml</filename> as the input
+       file for the <systemitem>psql2mysql</systemitem> tool. This is the
+       recommended way of running the tool.
+      </para>
+      <para>
+       Running the tool on the &admnode; is advisable if the input batch file
+       is already located there, or if you prefer not to install additional
+       software on cluster nodes.
+      </para>
+     </step>
+     <step>
+      <para>
+       To determine whether the &postgres; databases contain data that cannot
+       be migrated to &mariadb;, run <command>psql2mysql</command> with the
+      <literal>precheck</literal> option:</para>
+<screen>psql2mysql \
+--source postgresql://keystone:vZn3nfxXzv97@192.168.243.87/keystone \
+--target mysql+pymysql://keystone:vZn3nfxXzv97@192.168.243.88/keystone?charset=utf8 \
+precheck</screen>
+      <para>
+       To run precheck operation on all databases in a single operation, use the
+       <literal>--batch</literal> option and the
+       <filename>/etc/pg2mysql/databases.yaml</filename> file as follows:
+      </para>
+<screen>psql2mysql --batch /etc/pg2mysql/databases.yaml precheck</screen>
+      <para>
+       If the precheck indicates that there is data that cannot be imported
+       into &mariadb;, modify the offending data manually to fix
+       the problems.
```

Comment:
Should we add screenshot here to show how it looks if such an issue appears?

Reply:
Good idea. I'll try to create such output

Reply:
```# psql2mysql --source postgresql://cinder:idRll2gJPodv@192.168.243.86/cinder precheck
Table 'volumes' contains 4 Byte UTF8 characters which are incompatible with the 'utf8' encoding used by MariaDB
The following rows are affected:
+-----------------------------------------+-----------------+-------+
|               Primary Key               | Affected Column | Value |
+-----------------------------------------+-----------------+-------+
| id=5c6b0274-d18d-4153-9fda-ef3d74ab4500 |   display_name  |   💫   |
+-----------------------------------------+-----------------+-------+
Error during prechecks. 4 Byte UTF8 characters found in the source database.```